netmanager/src/config/constants.js

```diff
@@ -65,6 +65,8 @@ const prodConfig = {
   "http://34.78.78.202:30002/api/v1/devices/delete/components?comp=",
   UPDATE_COMPONENT_URI: 
   "http://34.78.78.202:30002/api/v1/devices/update/components?device=",
+  GET_online_offline_maintenance_status: 
```

Comment:
It would look nice if we follow the tradition of using upper cases for constants :)

Reply:
Done